libia2/include/ia2_compartment_init.inc

```diff
@@ -25,6 +25,11 @@
 #define STR(x) #x
 
 __thread void *COMPARTMENT_IDENT(ia2_stackptr) __attribute__((used));
+#if IA2_DISABLE
+void COMPARTMENT_IDENT(ia2_setup_destructors)(void) { }
```

Comment:
TODO: Add comment on why these need to be defined when compartments are disabled.